src/sidebar/store/modules/annotations.js

```diff
@@ -241,6 +236,8 @@ function addAnnotations(annotations) {
     });
 
     // If we're not in the sidebar, we're done here.
+    // FIXME Split the annotation-adding from the anchoring code; possibly
```

Comment:
Noted because this is the last dependency within the `annotations` store module on other store modules.